beetsplug/convert.py

```diff
@@ -156,6 +157,12 @@ def commands(self):
                               help=u'set the target format of the tracks')
         cmd.parser.add_option('-y', '--yes', action='store_true', dest='yes',
                               help=u'do not ask for confirmation')
+        cmd.parser.add_option('-l', '--link', action='store_true', dest='link',
+                              help=u'symlink files that do not need \
+                                     transcoding, instead of copying them; \
+                                     this may cause files in your library to \
+                                     be modified, if options such as "embed" \
+                                     are enabled')
```

Comment:
The in-line help string should usually be shorter than this—just one phrase is fine. We can leave the full detail for the documentation.